include/mull/Mutators/AndOrReplacementMutator.h

```diff
@@ -32,7 +32,9 @@ class MutationPointAddress;
 
 class AndOrReplacementMutator : public Mutator {
 
-  AND_OR_MutationType
+  AND_OR_MutationType findPossibleMutation(Value &V);
+
+    AND_OR_MutationType
   findPossibleMutationInBranch(BranchInst *branchInst,
```

Comment:
Seems like formatting is not correct.

Did you format the whole patch?

Reply:
Yes, I did. This one was from my cat 😸

Reply:
Weird, GitHub renders line 37 with indentation of 4 spaces and line 38 with 2 spaces.